src/lib/Scenes/Artwork/Components/__tests__/CommercialInformation-tests.tsx

```diff
@@ -1,16 +1,16 @@
 import { Sans, Theme } from "@artsy/palette"
 import { mount } from "enzyme"
 import { ArtworkFixture } from "lib/__fixtures__/ArtworkFixture"
+import { Countdown } from "lib/Components/Bidding/Components/Timer"
 import "moment-timezone"
 import React from "react"
 import { TouchableWithoutFeedback } from "react-native"
 import { ArtworkExtraLinks } from "../ArtworkExtraLinks"
-import { AuctionCountDownTimer } from "../AuctionCountDownTimer"
 import { BidButton } from "../CommercialButtons/BidButton"
 import { BuyNowButton } from "../CommercialButtons/BuyNowButton"
 import { CommercialButtons } from "../CommercialButtons/CommercialButtons"
 import { CommercialEditionSetInformation } from "../CommercialEditionSetInformation"
-import { CommercialInformation } from "../CommercialInformation"
+import { CommercialInformationTimerWrapper } from "../CommercialInformation"
```

Comment:
For future reference, it might have read nicer to rename this locally to just `CommercialInformation`.

```suggestion
import { CommercialInformationTimerWrapper as CommercialInformation } from "../CommercialInformation"
```